manifests/init.pp

```diff
@@ -355,6 +359,7 @@
   $proxy_password            = $pulp::params::proxy_password,
   $num_workers               = $pulp::params::num_workers,
   $enable_crane              = $pulp::params::enable_crane,
+  $max_tasks_per_child       = $pulp::params::max_tasks_per_child,
```

Comment:
Would you mind calling ```validate_integer``` on this as well? Then we know it's an int and you can leave out the ```.to_i``` in the templates.